source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,39 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+    template<typename Policy, typename InputKeyIt, typename InputValueIt,
+        typename Comparator =
+            std::less<typename std::iterator_traits<InputIt>::value_type>>
+    void
+    sort_by_key(Policy&& policy, InputKeyIt keys_first, InputKeyIt keys_last,
+                InputValueIt values_first, Comparator comp =
+                    std::less<typename std::iterator_traits<InputKeyIt>::value_type>());
```

Comment:
In the algorithm description below, `std::less{}` is said to be the default comparator. However, since C++14 it means `std::less<void>{}`, which is inconsistent with how the signature is specified. I suggest it to be `std::less` without explicit template argument everywhere. If necessary, the implementation may need to be adjusted accordingly.

Reply:
Let's keep it consistent for now, and then change everything in a separate PR.